.pre-commit-config.yaml

```diff
@@ -1,5 +1,6 @@
 repos:
   - repo: https://github.com/gruntwork-io/pre-commit
-    rev: v0.0.2
+    rev:  v0.1.10
     hooks:
       - id: terraform-fmt
+      - id: gofmt
```

Comment:
NIT: We should switch to `goimports` at some point, which organizes the imports better by extracting out the third party libs from stdlib. It also runs `gofmt`.

Reply:
Agreed. Can take a pass at it as part of TF 0.14 or AWS Provider 4.x 😁